### PR TITLE
Fix start service repeat sequence

### PR DESCRIPTION
**Pull request by Anatoliy-Leshin**
Fix retry sequence for NaviStartStreaming/AudioStartStream requests.
    
Issue: APPLINK-24863

### CODE REVIEW

**Comment by Anatoliy-Leshin** [comment]
@vlantonov, @anosach-luxoft, @AGritsevich, Please, review.

**Comment by vlantonov** [comment]
@Anatoliy-Leshin Reviewed

**Comment by AGritsevich** [comment]
@Anatoliy-Leshin, your build failed by style issues. Check it, please.
And I recommend you to install pre-commit hook:
```
## Pre-commit hook installation
* Go to tools/infrastructure/
* Run install-hooks.py
* or : Just run "python tools/infrastructure/install-hooks.py"
```

File: src/components/application_manager/src/commands/hmi/navi_start_stream_request.cc [hunk 1]
```diff
@@ -108,8 +108,10 @@ void NaviStartStreamRequest::on_event(const event_engine::Event& event) {
         break;
       }
       if (hmi_apis::Common_Result::REJECTED == code) {
-        LOGGER_INFO(logger_, "StartStream response REJECTED ");
-        SendRequest();
+        LOG4CXX_INFO(logger_, "StartStream response REJECTED ");
```
Comment (pestOO) [inline src/components/application_manager/src/commands/hmi/navi_start_stream_request.cc]: Amy be warning?

File: src/components/application_manager/src/commands/hmi/navi_start_stream_request.cc [hunk 2]
```diff
@@ -143,25 +144,24 @@ void NaviStartStreamRequest::RetryStartSession() {
   }
 
   if (app->video_streaming_approved()) {
-    LOGGER_INFO(logger_,
-                "NaviStartStream retry sequence stopped. "
-                    << "SUCCESS received");
+    LOG4CXX_DEBUG(logger_,
```
Comment (pestOO) [inline src/components/application_manager/src/commands/hmi/navi_start_stream_request.cc]: Warning?

Comment (Anatoliy-Leshin) [inline src/components/application_manager/src/commands/hmi/navi_start_stream_request.cc]: No, it's expected behaviour.

File: src/components/application_manager/src/commands/hmi/navi_start_stream_request.cc (hunk 1 above)
Comment (Anatoliy-Leshin) [inline src/components/application_manager/src/commands/hmi/navi_start_stream_request.cc]: Usual case, when user presses cancel on pop-up

File: src/components/application_manager/src/commands/hmi/navi_audio_start_stream_request.cc [hunk 3]
```diff
@@ -96,7 +96,7 @@ void AudioStartStreamRequest::on_event(const event_engine::Event& event) {
               message[strings::params][hmi_response::code].asInt());
 
       if (hmi_apis::Common_Result::SUCCESS == code) {
-        LOGGER_INFO(logger_, "StartAudioStream response SUCCESS");
+        LOG4CXX_INFO(logger_, "AudioStartStream response SUCCESS");
```
Comment (anosach-luxoft) [inline src/components/application_manager/src/commands/hmi/navi_audio_start_stream_request.cc]: @Anatoliy-Leshin we have no more `LOG4CXX_INFO` macro in `SDL`

**Comment by Anatoliy-Leshin** [comment]
@AGritsevich, @anosach-luxoft, please, review latest updates. Anton can you be merger this time?

File: src/components/application_manager/src/commands/hmi/navi_audio_start_stream_request.cc (hunk 3 above)
Comment (Anatoliy-Leshin) [inline src/components/application_manager/src/commands/hmi/navi_audio_start_stream_request.cc]: @anosach-luxoft 12253d802dec736d78162db78e87d133b2d25261

**Comment by AGritsevich** [comment]
@Anatoliy-Leshin Reviewed and Ok.

**Comment by Anatoliy-Leshin** [comment]
Thank you, @AGritsevich

**Comment by anosach-luxoft** [comment]
Reviewed

**Pull request merged**